\documentclass[9pt,lineno]{elife}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% font size %%%%%%%%%%%%%%%%%%%%%%%%
\usepackage{anyfontsize}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Maths %%%%%%%%%%%%%%%%%%%%%%%%%%%%
\usepackage{amsmath}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% referencing %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\usepackage{natbib}
\usepackage{hyperref}
\usepackage{xcolor}
\hypersetup{
    colorlinks,
    %linkcolor={red!50!black},
    linkcolor={black},
    citecolor={blue!50!black},
    urlcolor={blue!80!black}
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% figure and table %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\usepackage{graphicx}
\graphicspath{{./figures/}}
%\graphicspath{{}}
\usepackage[label font=bf,labelformat=simple, position = top]{subfig}
\newdimen\figrasterwd
\figrasterwd\textwidth
\usepackage{wrapfig}
\usepackage{lipsum}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% layout %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\usepackage{setspace}
\linespread{1.5}
\usepackage{authblk}

%\usepackage{fancyhdr}
\providecommand{\keywords}[1]{\textbf{\textit{Key words:}} #1}

%\newcommand\@shorttitle{}
% define \theshorttitle to what is given
%\newcommand\shorttitle[1]{\renewcommand\@shorttitle{#1}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% todo %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\usepackage[colorinlistoftodos]{todonotes}
\usepackage[disable]{todonotes}

\newcounter{todocounter}
\newcommand{\todonum}[2][]
{\stepcounter{todocounter}\todo[#1]{\thetodocounter: #2}}

\newcommand{\done}[2][]
{\todo[color=green!40, #1]{#2}}
\newcommand{\donenum}[2][]
{\stepcounter{todocounter}\done[#1]{\thetodocounter: #2}}


\newcommand{\narrative}[2][]
{\todo[color=blue!40, #1]{#2}}
\newcommand{\narrativenum}[2][]
{\stepcounter{todocounter}\narrative[#1]{\thetodocounter: #2}}

\newcommand{\donenarrative}[2][]
{\todo[color=green!40, #1]{#2}}
\newcommand{\donenarrativenum}[2][]
{\stepcounter{todocounter}\narrative[#1]{\thetodocounter: #2}}



\begin{document}

\title{Characterising local epidemiology of {\it P. falciparum} malaria through the structure of mixed infections}
\newcommand\shorttitle{Mixed infections in malaria}
\date{}

\author[?]{(LIST OF NAMES AND ORDER)}
\author[1]{Sha Joe Zhu}
\author[1,2,3,4]{Jacob Almagro-Garcia}
\author[?]{Pf3k Consortium}
\author[1,3]{Gil McVean}

\corr{gil.mcvean@bdi.ox.ac.uk}{GM}

\affil[1]{Big Data Institute, Li Ka Shing Centre for Health Information and Discovery, University of Oxford, Oxford, UK}
\affil[2]{Wellcome Centre for Human Genetics, University of Oxford, Oxford, UK}
\affil[3]{Medical Research Council (MRC) Centre for Genomics and Global Health, University of Oxford, Oxford, UK}
\affil[4]{Wellcome Trust Sanger Institute, Hinxton, UK}
%\donenum[inline]{updated affiliation}
%}

\maketitle{}

\begin{abstract}
Individuals infected with {\it Plasmodium falciparum} malaria pathogens can carry multiple distinct, though sometimes related, strains.  However, how the rate and relatedness of such mixed infections relates to local epidemiological parameters remains unclear.  Here, we develop an enhanced method for strain deconvolution from genome sequencing data, which estimates the number of strains, their proportions, relatedness profiles and individual haplotypes.  We validate the method through simulation and controlled experiments and apply it to 2,512 isolates from 14 countries.  Rates of mixed infection vary from 18\%-63\% across countries and correlate with independent estimates of prevalence (Pearson $r = 0.76,~p = 6.1e-07$).  54\% of mixed infections involve more than two strains and 71\% include sibling strains likely to have to be have been co-transmitted from a single mosquito.  Relatedness among strains decreases with prevalence ($r = -0.72,~p = 5.1e-06$) and differs between continents, with Asian isolates typically showing elevated relatedness compared to African isolates.  We develop a mathematical model that shows how characteristics of mixed infection relate to local epidemiological process and propose that monitoring spatial and temporal patterns of mixed infection will be highly informative in pathogen surveillance.
\end{abstract}

\keywords{Malaria, genome, epidemiology, relatedness}


\section{Introduction}


Individuals infected with malaria-causing parasites of the genus {\it Plasmodium} can often be found to carry multiple, distinct strains~\citep{Bell2006}  Such mixed infections are likely indicative of intense local exposure rates, being found commonly in regions of Africa with high rates of prevalence~\citep{Bhatt2015, Howes2016}, though are also documented for {\it P.~vivax} and other malaria-causing parasites~\citep{Mueller2007, Collins2012}, even in regions of much lower prevalence~\citep{Howes2016, Steenkeste2010}.  For example, \citet{Pearson2016} reports 45\% of {\it P.~vivax} infections are mixed.  Mixed infections have been associated with increased disease severity \citep{deRoode2005} and also provide the fuel for generating genomic diversity within the parasite, enabling co-transmission to the mosquito vector where sexual recombination occurs \citep{Mzilahowa2007}.  Mixed infections are transient \citep{Bruce2002, Zimmerman2004}, but little is known about distribution of duration or whether clearance of one or more strains results purely from host immunity~\citep{Borrmann2011} or can be influenced by interactions between the distinct strains is not known~\citep{Enosse2006, Bushman2016}.

Although mixed infections can be studied from SNP bar codes~\citep{Galinsky2015} or single locus polymorphisms~\citep{Jack2016}, genome sequencing provides a powerful approach for detecting mixed infections~\citep{Chang2017}.  Genetic differences between strains within a host are manifest as polymorphic sites within the isolate, which motivates statistical approaches for estimating the number of distinct strains, their relative proportions and genome sequences~\citep{Zhu2017}.  Although genomic approaches cannot identify individuals infected multiple times by identical strains (and have to filter sequencing data heavily to eliminate false signs of polymorphism arising from sequencing error and problems of incomplete or erroneous reference assemblies), they provide a rich characterisation of within host diversity~\citep{Manske2012}.

An important observation made from sequencing data collected to date is that the different strains present can be highly related~\citep{Nair2014, Trevino2017}.  For example, in {\it P. vivax}, 58\% of mixed infections show long stretches of within host homozygosity have been observed~\citep{Pearson2016}; \citet{Nkhoma2012} reported an average of 78.7\% {\it P. falciparum} alleles sharing in Malawi and 87.6\% sharing in Thailand. This could result from co-infection by sibling strains, where a mosquito vector had acquired multiple strains from biting multiple infected hosts, hosts themselves multiply infected, or both.  Alternatively, relatedness can occur when population diversity is low, such as during the early stages of an outbreak or following severe population bottlenecks.  At the population level, extensive relatedness has also been observed.  For example, following the impact of control programmes within Senegal~\citep{Mouzin2010}, haplotype diversity within isolates dropped as a result of the population bottleneck~\citep{Wong2017}.

The rate and relatedness structure of mixed infections (and how these relate to other infected individuals within the sample and/or the population) clearly relate to the local epidemiology.  However, progress towards utilising this source of information is limited by three problems.  The first is that, while strain deconvolution within mixed infections has received substantial attention~\citep{Galinsky2015, Jack2016, Chang2017}, joint deconvolution of strains and estimation of relatedness has not been attempted.  Because existing deconvolution methods assume equal relatedness along the genome, differences in relatedness that occur, for example, through infection by sibling strains can lead to errors in the estimation of the number, proportions and sequences of individual strains (Figure \ref{fig:fig1}).  Recently, progress has been made in the case of dual-infections with balanced proportions~\citep{Henden2016}, but a general solution is lacking.  The second limitation is that little is known about how the rate and relatedness structure of mixed infections relates to underlying epidemiological parameters.  Informally, mixed infections will occur when prevalence is high; an observation exploited by \citet{Cerqueira2017} when estimating changes in transmission over time.  However, the quantitative nature of this relationship, the key parameters that influence mixed infection rates and how patterns of relatedness relate to infection dynamics are largely unexplored.  Finally, an important, but often overlooked issue is the sampling design.  Malaria parasites may be taken from individuals presenting with disease or as part of a surveillance programme.  They are also often highly clustered in time and space.  What the impact of different approaches have on observed genomic variation is not clear.  Nevertheless, because mixed infection rates are likely to respond rapidly to changes in prevalence, exploring these issues is an important task.

Here, we develop, test and apply an enhanced method for strain deconvolution, which builds on the previously-published \texttt{DEploid} software.  The method works by separating the process of estimating strain number, proportions and relatedness (specifically identity-by-descent, or IBD, profile along the genome) from the problems of inferring genome sequences and provides substantial advances in accuracy in complex setting or lower coverage data.  We apply the approach to 2,512 isolates of {\it P. falciparum} drawn from 14 countries and a range of years, characterising the rate and nature of mixed infections, including patterns of relatedness.  We show that rates of mixed infection are highly correlated with independent, non-genetic estimates of malaria infection prevalence, and that levels of IBD are negatively correlated with mixed infection rates, hence prevalence,, but also influenced by other aspects of local epidemiology, such as recent dramatic changes in transmission. For example, we observe a lower number of effective parasite strains within isolates in Senegal compare to other Africa countries.  We introduce a simple deterministic mathematical model that identifies key epidemiological parameters in determining statistics of mixed infection and that recapitulates several empirical observations.  Finally, we discuss the likely impact of sampling strategies and the prospects for using pathogen genomic data to augment surveillance within control programmes.



\section{Strain deconvolution in the presence of relatedness}

Existing methods for deconvolution of mixed infections assume that the different genetic strains present in mixed infections are unrelated.  This assumption allows for efficient computation of priors for allele frequencies within samples, either through assuming independence of loci~\citep{Jack2016} or as sequences generated as imperfect mosaics of some (predefined) reference panel~\citep{Zhu2017}.  However, when strains are related to each other, certain allele frequencies are excluded; for example, within a region of IBD for a mixed infection with two strains, only allele frequencies of 0 and 1 are possible.  Because patterns of IBD are likely to vary along the genome, these constraints can cause problems for estimators, which can try to fit complex strain combinations (with relatedness) as simpler configurations (without relatedness).  Below we outline the approach taken to integrating IBD into \texttt{DEploid} and validate the approach.  Further details are provided in the Supplementary Material.


\subsection{Decoding genomic relatedness among strains}

Many approaches to detecting IBD using hidden Markov models of transition into and out of IBD~\citep{Chang2015, Gusev2009, Gusev2011}.  We have generalised this approach for the case of $k$ strains, where there are $2^k$ possible genotype configurations at any one locus ({\it Plasmodium} strains are haploid).  For a given set of strain proportions within the isolate, this results in $2^k$ possible distinct allele frequencies; for example, the isolate shown in Figure~\ref{fig:fig1}a has three strains, hence eight possible allele frequencies.  The IBD state is defined by the sharing patterns among strains.  For example, with two strains there are only the configurations $\{1,2\}$ and ${1,1}$, corresponding to non-IBD and IBD states respectively.  With three strains there are five states: $\{1,2,3\}$, $\{1,1,2\}$, $\{1,2,1\}$, $\{1,2,2\}$ and $\{1,1,1\}$, four strains have 15 IBD configurations, etc.  We restrict analyses to a maximum of four strains for reasons of computational efficiency, though in practice we have not identified any samples where five distinct strains can be identified.  The effect of IBD is to restrict the number and type of genotype configurations that are possible.  Recombination breaks up IBD, hence allele frequency patterns, along the genome (Figure~\ref{fig:fig1}b).  For inferring patterns of IBD we assume linkage equilibrium between variants, which speeds up computation, and use a Gamma-Poisson emission model for read counts (see Supplementary Methods).  Allele frequencies are estimated from a panel of reference isolates obtained from a similar geographic region.  Because of the structure of the hidden Markov model, we can compute the likelihood of the strain proportions by integrating over all IBD configurations.  Once the number and strain proportions have been estimated, we can then use maximum likelihood and posterior decoding to infer the relatedness structure across the genome (Figure~\ref{fig:fig1}b).

Unlike our previous work, \texttt{DEploidIBD} infers strain structure in two steps.  In the first we estimate the number and proportions of strains using MCMC, allowing for IBD as described above.  In the second, we infer the individual genomes of the strains, using the MCMC methodology of~\citet{Zhu2017}, which can account for linkage disequilibrium (LD) between variants, but without updating strain proportions.


\begin{figure}[ht]
  \begin{center}
    \includegraphics[width=\textwidth]{Fig1.pdf}
    \caption{Deconvolution of field sample PD0577-C from Thailand.  (a) Summary of allele frequencies observed within the isolate shown as a scatter-plot of the numbers of reads supporting the reference (REF: x-axis) and alternative (ALT: y-axis) alleles. (b) The profile of within-sample allele frequency along chromosomes 11 and 12 (black points) suggests a changing profile of IBD with three distinct strains, estimated to be at frequencies of $22\%$, $52\%$ and $26\%$ respectively (other chromosomes omitted for clarity, see Figure~\ref{fig:fig1}-Supplement~1); blue points indicate expected allele frequencies within the isolate.  However, the strains are inferred to be siblings of each other: green segments identify where all three strains are IBD; yellow, orange and dark orange segments identify the regions where one pair of strains are IBD but the others are not.  In no region are all three strains inferred to be distinct. A graphical description of the modules and work flows for \texttt{DEploidIBD} is given in Figure~\ref{fig:fig1}-Supplement~2.}\label{fig:fig1}
  \end{center}
   \figsupp{Whole genome deconvolution of field sample PD0577-C is represented in ring figures. The outer ring show expectedwithin-sample allele frequency (WSAF) (blue) and observed WSAF (red) across the genome. Red and blue points indicate observed and expected allele frequencies within the isolate. This inner ring marks the IBD regions of the three strains: green segments identify where all three strains are IBD; yellow, orange and dark orange segments identify the regions where one pair of strains are IBD but the others are not.  In no region are all three strains inferred to be distinct, suggesting that all three strains are siblings.}{\includegraphics[width=.7\textwidth]{{myring.ring}.pdf}}
   \figsupp{A graphical description of the modules and work flows for \texttt{DEploidIBD}. The purple boxes at the bottom represent final outputs of the pipeline. The black boxes indicate when \texttt{DEploidIBD} is executed, with inputs highlighted by blue boxes. The process can be explained in 3 steps: 1. Apply \texttt{DEploidIBD} with population level allele frequencies to infer clonal haplotypes. 2: Apply \texttt{DEploidIBD} to infer the number of strains and strain proportions. 3: Apply \texttt{DEploidIBD} with fixed the proportions from step 2, and use haplotypes from step 1 as the reference panel for deconvolution.
   }{\includegraphics[width=.8\textwidth]{scheme.pdf}}
\end{figure}


\subsection{Validation}

We validated \texttt{DEploidIBD} through two experiments.  First, to test consistency with \texttt{DEploid} \citet{Zhu2017}, we re-analysed the 27 experimental mixtures from \citep{Wendler2015}.  This data set includes 27 samples of various mixtures of four laboratory parasite lines (3D7, Dd2, HB3 and 7G8; Figure~\ref{fig:benchmark} – figure supplement 1).  Allowing for mixtures of up to four strains and using optimal reference panels, we found comparable performance with the single-step \texttt{DEploid} method, with the exception of three strains of equal proportions where LD information is necessary to achieve accurate deconvolution (Figure~\ref{fig:benchmark}a).

To test the performance of \texttt{DEploidIBD} in a more realistic setting, we created {\it in silico} mixtures from 212 clonal samples of Asian origin with two proportions (25/75\% and 45/55\%) for 8,071 sites from Chromosome 14.  A further 20 randomly chosen samples were used as the reference panel. In order to compare the performance of the two methods at different levels of relatedness, we set 25\%, 50\% and 75\% of the second haplotype to be the same as the first haplotype to mimic scenarios of low, medium and high relatedness. To simulate data, we used empirical read depths and drew read counts for the two alleles from binomial proportions.  We inferred strain proportions (summarised by the effective number of strains: $1/\sum w_{i}^{2}$), and haplotypes.  Both \texttt{DEploid} and \texttt{DEploidIBD} correctly estimate strain proportions with low to moderate relatedness and high coverage.  However, for samples with closely related strains, particularly at low coverage, only \texttt{DEploidIBD} can identify the correct number of strains and their proportions.  Phasing switch errors are comparable between the two methods and decrease with increased relatedness due to the higher homozygosity.  Rates of genotype error are also similar for the two approaches in settings of low relatedness (error rate of 0.4\% per site for 25/75 mixtures and 1.0\% for 45/55 mixtures).  However, for the 25/75 mixtures with high relatedness, genotype error for the non-IBD approach increases to 0.6\%, while error in the IBD approach remains at 0.4\%.  In summary, joint inference of IBD profiles and strain haplotypes is expected to improve estimates of strain proportions (and hence haplotypes), particularly in regions with high rates of IBD and with low sequencing coverage.  Moreover, direct estimates of IBD within mixed infections can be used as additional characterisation of samples.



\begin{figure}[htp]
  \begin{center}
  \includegraphics[width=\textwidth]{Fig2.pdf}
  \caption{Simulation results. (a) Experimental validation and effective number of strains inferred by {\tt DEploidIBD}. We use a reference panel of the laboratory parasite lines (3D7, Dd2, HB3 and 7G8) to deconvolve the same lab-mixed samples, assuming 3 and 4 strains within a sample. Each experiment is performed with and without IBD inference. Black crosses indicate the true effective number of strains. Purple crosses indicate median values obtained from 30 replicates when using {\tt DEploidIBD} with a panel and assuming that the maximum number of strains is 4. The coloured dots show the inferred effective number of strains across replicates with intensity proportional to fraction. Note one sample (indicated by an asterisk) where balanced proportions of three strains results in the LD-free approach underfitting the data as a mixture of two strains with proportions of $1/3$ and $2/3$.\\  (b) Comparison of effective number of strains between \texttt{DEploid} and \texttt{DEploidIBD}. The effective number of strains is expected to be 1.6 for $25/75$ mixtures and 1.98 for $45/55$ mixtures. \texttt{DEploidIBD} can correctly recover this value when two strains are highly related, where \texttt{DEploid} fails to do so. (c) Cumulative frequencies of average per site genotype error across 76 simulated mixtures with three levels of IBD (25\%, 50\% and 75\%) using empirical data from clonal samples. We find higher genotype error rate in \texttt{DEploid} than \texttt{DEploidIBD}, when inferred proportions are incorrect. In figures (b) and (c), we excluded eight cases out of the 100 experiments where simulated haplotypes were over 99\% identical, and 16 cases where average coverage was below 20.
}\label{fig:benchmark}
  \end{center}
  \figsupp{Comparison of true and inferred haplotypes for Chromosome 14 (2,369 SNPs) in sample PG0396-C before and after haplotype painting using a reference panel of the laboratory parasite lines (3D7, Dd2, HB3 and 7G8). The yellow, cyan and white background label the haplotype segments from strains 7G8, HB3 and Dd2 respectively. The switch errors are obtained by counting the changes of a strain segment mapped to reference strains; the genotype errors are the discordance between the strain and the mapped reference segments. }{\includegraphics[width = 0.8\textwidth]{DEploid_IBD_haps_compare.pdf}}
\end{figure}




\section{Results}

\subsection{The rate and structure of mixed infection in 2,512 {\it P. falciparum} field samples from 14 countries}

To investigate how the rate and relatedness structure of mixed infections varies among geographical regions with different epidemiological characteristics, we applied \texttt{DEploidIBD} to 2,512 field samples of {\it P. falciparum} released by the \citet{pf3k}.  These samples were collected under a wide range of designs: for example, \citet{Amambua-Ngwa2012} recruited The Gambia patients during the malaria season from four health facilities located within a radius of 20 km from the coast; \citet{Ashley2014} recruited patients from an open-label, randomized Artemisinin clinical trial at 15 sites in 10 countries. The majority of samples were taken from individuals seeking treatment for malaria symptoms. These patients were either from specific clinical trials \citep{Ocholla2014} or under specific treatment \citep{Duffy2015, Miotto2013,eLife2016}.  A summary of the data sources is presented in Table~\ref{tab:pf3k}.  After variant calling and filtering, we genotyped approximately 1M variable SNP positions across all samples, sequenced to an average depth of 86 (range 1.0 – 680).  See Supplementary Material for details of filters and data availability.  For deconvolution, samples were grouped into four geographical regions within Africa and three within Asia based on genetic similarity (Table~\ref{tab:Pf3k}).  Diagnostic plots for all samples can be found in \url{https://github.com/mcveanlab/mixedIBD-Supplement}.  Information on the availability of inferred genome sequences is provided below.

\begin{table}[btp]
  \caption{Summary of African samples.}\label{tab:Pf3k}
{\small
\begin{tabular}{p{1.5cm} c c c |p{1.5cm} c c c c c}
\toprule
        &          &&&         & Sample & Mean read             & & &\\
Country &   Year   &$PfPr_{0}$&$PfPr$&Location & size   & depth (s.e.)  & Relatedness & Effect K&\\
\midrule
Gambia          &2008           &0.14  &0.15  &Brikama&65   &129  (9.4 )&0.53 &1.3  &\\
%\citet{Amambua-Ngwa2012}&          &&&         &  &             & & &\\
\hline
Guinea          &2011           &0.39  &0.47  &Nzerekore&100  &75   (4.5 )&0.22 &1.4  &\\
%\citet{eLife2016}&          &&&         &  &             & & &\\
\hline
Ghana           &2009           &0.31  &0.33  &Navrongo&125  &85   (5.6 )&0.25 &1.6  &\\
                &2010           &0.31  &0.34  &               &187  &117  (9.7 )&0.26 &1.5  &\\
                &2011           &0.31  &0.36  &               &97   &76   (5.3 )&0.26 &1.5  &\\
                &               &               &               &Kintampo&14   &78   (7.9 )&0.18 &1.4  &\\
                &2012           &0.32  &0.39  &               &50   &137  (9.4 )&0.27 &1.7  &\\
                &               &               &               &Navrongo&48   &109  (4.4 )&0.35 &1.7  &\\
                &2013           &0.33  &0.41  &Kintampo&4    &172  (38.4)&0.53 &1.1  &\\
                &               &               &               &Navrongo&92   &114  (4.5 )&0.3  &1.6  &\\
\hline
Mali            &2007           &0.27  &0.28  &Kolle&51   &82   (10.5)&0.36 &1.6  &\\
                &               &               &               &Faladje&36   &75   (10.1)&0.34 &1.3  &\\
                &               &               &               &Bandiagara&9    &95   (25.2)&0.39 &1.8  &\\
\hline
Malawi          &2011           &0.33  &0.36  &Chikwawa&317  &98   (2.4 )&0.27 &1.7  &\\
                &               &               &               &Zomba&52   &76   (7.1 )&0.29 &1.7  &\\
\hline
Congo DR        &2013           &0.29  &0.33  &Kinshasa&113  &49   (3.2 )&0.36 &1.5  &\\
\hline
Nigeria         &2012           &0.5   &0.51  &Ilorin&4    &51   (20.9)&0.62 &1.4  &\\
\hline
Senegal         &2004           &0.25  &0.26  &Thies&2    &130  (68.2)&0.03 &1.4  &\\
                &2009           &0.14  &0.19  &               &43   &175  (14.9)&0.47 &1.1  &\\
                &2010           &0.1   &0.16  &               &24   &159  (9.7 )&0.36 &1.3  &\\
                &2011           &0.06  &0.13  &               &32   &97   (6   )&0.4  &1.1  &\\
\hline
\hline
Thailand        &2011           &0.05  &0.06  &Mae Sot&37   &67   (7.3 )&0.36 &1.2  &\\
                &               &               &               &Sisakhet&5    &112  (25.4)&0.17 &1.3  &\\
                &2012           &0.05  &0.06  &Mae Sot&69   &83   (4.9 )&0.59 &1.3  &\\
                &               &               &               &Sisakhet&13   &89   (13  )&0.37 &1.1  &\\
                &               &               &               &Ranong&11   &82   (12.4)&0.55 &1.2  &\\
                &2013           &0.04  &0.05  &Sisakhet&4    &57   (7.8 )&0.09 &1.2  &\\
\hline
Cambodia        &2009           &0.01  &0.02  &Pursat&19   &75   (8.8 )&0.39 &1.3  &\\
                &2010           &0.01  &0.02  &               &105  &95   (6.8 )&0.65 &1.2  &\\
                &               &               &               &Ratanakiri&50   &71   (6.1 )&0.44 &1.3  &\\
                &2011           &0.02  &0.02  &Pursat&104  &48   (3   )&0.63 &1.2  &\\
                &               &               &               &Ratanakiri&82   &45   (4.2 )&0.47 &1.4  &\\
                &               &               &               &Preah Vihear&74   &51   (5.2 )&0.36 &1.2  &\\
                &               &               &               &Pailin&50   &53   (4.1 )&0.43 &1.1  &\\
                &2012           &0.02  &0.03  &Pursat&7    &37   (19.1)&0.58 &1.4  &\\
                &               &               &               &Preah Vihear&30   &43   (6.7 )&0.57 &1.1  &\\
                &               &               &               &Ratanakiri&15   &44   (8.9 )&0.32 &1.3  &\\
                &               &               &               &Pailin&34   &46   (5.2 )&0.52 &1.1  &\\
\hline
Bangladesh      &2012           &NA    &NA    &Ramu &50   &53   (4.2 )&0.49 &1.5  &\\
\hline
Viet Nam        &2011           &NA    &NA    &Phuoc Long&27   &68   (7.2 )&0.38 &1.2  &\\
                &               &NA    &NA    &Bu Gia Map&44   &66   (5   )&0.47 &1.3  &\\
                &2012           &NA    &NA    &               &20   &110  (8.7 )&0.51 &1.2  &\\
                &               &NA    &NA    &Phuoc Long&5    &107  (6.3 )&0.82 &1.2  &\\
\hline
Myanmar         &2011           &0.05  &0.05  &Bago &12   &59   (7.1 )&0.26 &1.2  &\\
                &2012           &0.05  &0.05  &               &47   &62   (5.2 )&0.46 &1.2  &\\
\hline
Laos            &2011           &0.17  &0.17  &Attapeu&59   &71   (4.2 )&0.37 &1.4  &\\
                &2012           &0.17  &0.17  &               &26   &77   (7   )&0.67 &1.3  &\\
\hline
\bottomrule
\end{tabular}
}

\tabledata{Summary of Pf3k samples in data release 5.1.  Genotyping, including both indel and SNP variants, was performed using a pipeline based on GATK best practices, see Methods. Data available from \url{ftp://ngs.sanger.ac.uk/production/pf3k/release_5/5.1}.}
\end{table}


Across continents and countries we find substantial variation in the rate and relatedness structure of mixed infections.  Within Africa, rates of mixed infection vary from 19\% in Senegal to 68\% in Malawi (Figure~\ref{fig:mixInfPlot}a).  In the South East Asian field samples, mixed infection rates are generally lower, though also vary considerably; between 28\% in Myanmar to 56\% in Bangladesh.  Where data for a location is available over multiple years, we find no evidence for significant fluctuation over time (though we note that these studies are typically not well powered to see temporal variation).  Between 5.1\% (Senegal) and 40\% (Malawi) of individuals have more than two strains present and, at the population level, there is a positive correlation (Pearson correlation coefficient = 0.64) between the rate of dual infection (where two strains are present) and higher levels of infection (weighted linear model P = 0.05, accounting for continental level effects).

\begin{figure}[h]
  \begin{center}
    \includegraphics[width=\textwidth]{Fig3.pdf}
    \caption{Properties of mixed infections across 2,512 field samples of {\it Plasmodium falciparum}. (a) The fraction of samples from each population inferred to have one to four distinct pathogens present.  Populations are ordered by rate of mixed infection within continent. (b) The spectrum of IBD within mixed infections (including dual, triple and quad infections), broken down into unrelated (where the fraction of the genome inferred to be IBD, $r$, is $< 0.1$), low IBD ($0.1 \geq r < 0.3)$, sib-level ($0.3 \geq r<0.7$) and high ($r \geq 0.7$).  Populations ordered as in part a. (c) The correlation between rates of mixed infection and level of identity by descent. Populations coloured by continent and shown $\pm 1$ s.e.m..  The dotted line shows the slope of the regression from a weighted lienar model excluding Senegal.  Abbreviations: SEN-Senegal, GAM-The Gambia, NIG-Nigeria, GUI-Guinea, DRC-The Democratic Republic of Congo, MALI-Mali, GHA-Ghana, MALA-Malawi, MYA-Myanmar, THA-Thailand, VIE-Vietnam, CAM-Cambodia, LAO-Laos.} \label{fig:mixInfPlot}
  \end{center}
\end{figure}


Relatedness between samples and populations also varies substantially.  In dual infections, the average fraction of the genome inferred to be IBD ranges from 33\% in Senegal to 65\% in Nigeria ((Figure~\ref{fig:mixInfPlot}b).   Asian populations show, on average, a higher level of relatedness within dual infections (56\%) compared to African populations (45\%).  Levels of IBD in samples with three or more strains are comparable to those seen in dual infections (average IBD being 0.56 in Asia and 0.49 in Africa) and significantly correlated (weighted linear model P = 0.014).  Overall, 64\% of all mixed infections involve strains with over 30\% of the genome IBD (the 2.5th percentile of IBD expected between true siblings given the recombination rate in {\it P. falciparum}).

Overall, we find that populations with higher rates of mixed infection tend to have lower levels of IBD in mixed infections (weighted linear model P = 0.037 after accounting for a continental level difference).  However, the continental level effect is driven by Senegal, which has an unusual combination of low mixed infections, but also low IBD.  Excluding this population (see below for addition discussion), we find a consistent pattern across populations (Figure~\ref{fig:mixInfPlot}c), with a strong negative effect of mixed infection rate on level of IBD (P = $4.2\times10^{-7}$).

The observed levels of IBD suggest that a substantial fraction of mixed infections involve sibling strains; those related through meiosis from distinct parental strains within a multiply-infected mosquito.  To further investigate this hypothesis we examined the length distribution of IBD segments within mixed infections and compared these to observations from sibling strains generated through experimental lab crosses.  Figure~\ref{fig:strainIBD} shows examples of genomic IBD profiles in dual and triple infections.  In lab crosses cross-over events occur typically 200~kb to 1~Mb apart \citet{Miles2016}, hence IBD tracts are expect to be about half this length in dual sibling infections and one third of this in triple infections (Figure~\ref{fig:strainIBD}).  By jointly considering IBD fraction and IBD length, we observe bimodal distributions of both variables, which can be used to identify mixed infections related to different degrees.  For example we can identify those likely to be siblings from unrelated parents, siblings from related parents (i.e. with inbreeding) and strains separated by two to three meioses (Figure~\ref{fig:strainIBD}).  For example, within dual infections, we estimated that 27\% are sib infections in Africa and 43\% are sib infections in Asia.


\begin{figure}[h]
  \begin{center}
  \includegraphics[width=\textwidth]{Fig4.pdf}
   \caption{Identifying sibling strains within mixed infections.  (a) Plots showing the ALT versus REF plots (left hand side) and inferred IBD profiles along the genome for five strains of differing composition.  From top to bottom: A dual infection of highly related strain s ($r=0.84$), a dual infection of two sibling strains ($r=0.6$), a triple infection of three sibling strains - note the absence of no stretches without IBD, a triple infection of two sibling strains and one unrelated strain, and a triple infection of three unrelated strains.  The numbers below the sample IDs indicate the average pairwise IBD, $r$, and the mean length of IBD segments, $l$, respectively.  (b) The joint (scatter-plot) and marginal (histograms) distributions of IBD fraction (x-axis) and median IBD fragment length (y-axis) within mixed infections for dual and triple infections.  Quad infections are omitted due to high levels of uncertainty. Points (and regions) are coloured by inferred class of relatedness ('unrelated', 'cousin'-level relatedness, 'sib-level' relatedness and 'super-sib' level, which are siblings arising from related parents); see Supplementary Methods for details.  The red line indicates a smoothed (loess) estimate of the relationship. The pale shaded area indicates the range of median IBD fragment lengths observed within experimental lab crosses of {\it P. falciparum}.}\label{fig:strainIBD}
   \end{center}
   \figsupp{IBD chunk lengths from crosses.}{\includegraphics[width=\textwidth]{Fig4Sup.pdf}}
\end{figure}



\subsection{Characteristics of mixed infection correlate with local pathogen prevalence}

To assess how characteristics of mixed infection relate to local infection intensity, we obtained estimates of {\it P. falciparum} prevalence from the Malaria Atlas Project (MAP).  MAP provides smoothed estimates of prevalence, as well as summaries of the individual studies that are used to build the map. The smoothed estimates of prevalence  range from 0.009\% in Bangladesh to 41\% in Guinea, with African countries having over two orders of magnitude greater average infection intensity (mean of 23\% in Africa and 0.15\% in Asia). However, seasonal and geographic fluctuations in prevalence mean that, conditional on sampling an individual with malaria, local prevalence may be much higher than the longer-term (and more geographically widespread).  We therefore estimated a conditional prevalence, by excluding reports with zero positive results (see Table~\ref{tab:Pf3k}).


because we can only measure malaria genomic data in individuals with malaria, we re-estimate prevalence excluding studies with an estimate of zero, which represents a sub


\citet{MAP2017} matched to year and country level.  Across the populations studied,  We summarise mixed infection rates by the average effective number of strains (see above), which reflects both the number and proportion of strains present.  This metric both avoids the problem of having to determine a threshold for determining the presence of a very low proportion strain and is sensitive to the presence of triply (and more) infected samples.

After accounting for between-continent differences in prevalence, the effective number of strains is a significant predictor of local prevalence (linear model P = $3.7 \times 10^{-4}$), though the level of relatedness is not ($P>0.49$), either by itself or in combination in the rate of mixed infection (Figure~\ref{fig:pfpr}).  The relationship between mixed infection rate and prevalence is  driven entirely by the African populations ($P = 0.0037$).  Within the Asian populations, mixed infection rate is an extremely poor predictor of country-level prevalence ($P = 0.51$). FEELS A BIT THIN.

\begin{figure}[h]
  \centering{}
    \includegraphics[width=\textwidth]{Fig5.pdf}
  \caption{The relationship between {\it P. falciparum} prevalence and characteristics of mixed infection.  The scatterplot shows, for each sampling location and year, the relationship between the average effective number of strains, given by $(\sum f_i^2)^{-1}$, where $f_i$ is the proportion of the $i$th strain, and the estimated country-wide average prevalence of malaria infection from the Malaria Atlas Project \citep{MAP2017}.  Point shape indicates country and colour indicates the average level of IBD observed within mixed infections. }
  \label{fig:pfpr}
\end{figure}

Two features stand out from this analysis.  First, despite having orders of magnitude difference in {\it P. falciparum} prevalence at the country level, rates of mixed infection in Asian populations are typically within a factor of two of those in African populations.  This observation is particularly surprising given the association between mixed infection rate and prevalence within African populations and could point to either substantial heterogeneity in local transmission rates within Asian populations or an unexpected consequence of malaria epidemiology.  Second, given the (negative) association between mixed infection rate and levels of IBD, it seems unexpected that levels of IBD do not correlate with prevalence at the country level.

To better understand the expected relationships between pathogen prevalence and statistics of infection, we developed a simple extension of the classic Ross-MacDonald model \citep{Smith2012} of malaria transmission to include mixed infection within both hosts and vectors (Figure~\ref{fig:model}). In this deterministic model, we assume a fixed number of hosts and vectors, that infection status does not affect probability of transmission or sampling and that the bottlenecks at transmission to host or vector can be approximated as a single parameter that determines the probability that a multiply-infected host (or vector) transmits more than one strain (if they are themselves multiply-infected.  See Supplementary Material for model details.  At equilibrium, this model recovers the standard relationship between vector density and/or biting rate and local prevalence: $P = (\lambda_H - \gamma \xi)/(\lambda_H + \lambda \xi)$, where $\lambda_H$ is the FILL IN PREFERRED VERSION OF EQUATION.  However, in addition, we  can predict how mixed infection rates covary with prevalence by changing the size of the vector population (hence biting rate) and approximate the relative contribution of sib-infections by comparing the contribution to novel mixed infections, at equilibrium, from previously uninfected hosts (hence infected by a single multiply-infected mosquito, assumed to transmit sibling strains, occurring at rate $H_0 V_M \lambda_H \phi$) and from previously singly-infected strains (hence carrying at least two distinct strains, occurring at rate $H_1 (V_1 + V_M) \lambda_H$).  This model has several limitations, for example not being able to model background levels of relatedness, but nevertheless serves as a useful indicator of how more complex and realistic models might behave.

Analysis of the model reveals two features.  First, by altering the vector population, the rate of mixed infection tracks prevalence in a largely linear manner, though at high prevalence, there is a somewhat greater than linear relationship, see Figure~\ref{fig:model}c.  Second, the relative contribution of sibling strains (compared to unrelated strains) in mixed infections is relatively constant across varying prevalence, with the exception of a  high contribution at very low prevalence (Figure~\ref{fig:model}c).  These observations are consistent with the genomic data presented here (Figure~\ref{fig:model}), where we see significant correlation between prevalence and mixed infection, but no significant correlation with relatedness in mixed infections (despite the negative correlation between mixed infection rate and IBD fraction).  Figure~\ref{fig:model}c shows a choice of parameter values that broadly match the empirical data.  These results suggest that rates of mixed infection are likely to be highly informative about the local intensity of {\it P. falciparum} transmission.  In contrast, the relative contribution of sibling infection to mixed infections reflects fundamental parameters relating to transmission events.


\begin{figure}[h]
  \centering{}
  \includegraphics[width=\textwidth]{Fig6.pdf}
  \caption{Deterministic model for mixed infection.  (a) Summary of the dynamical system used to model mixed infection rates in the host (left hand side) and vector (right hand side).  Quantities in circles (diamonds) refer to numbers of uninfected (subscript 0), singly infected (subscript 1) and multiply infected (subscript M) hosts (vectors).  Bold circles indicate where samples are taken from (assumed to be independent of multiplicity of infection).  Parameters governing the rate of transition are indicated along arrows: $\lambda_H$ = number of bites per individual vector per day that could result in a host infection event; $\phi$ = probability that an infectious bite from a multiply-infected vector establishes a multiple infection in a host; $\alpha$ = rate, per day, at which a multiply-infected host clears to a singly infected host; $\gamma$ = rate at which a host clears all infections present; $\lambda_V$ = number of times an individual vector bites a given host per day in which the vector could become infected by pathogen present in the host; $\theta$ = probability that a vector, when being infected through biting a multiply-infected host, becomes multiply-infected; $\xi$ = rate, per day, at which vectors clear pathogen (equivalent to the rate of vector replacement per day).  (b) Comparison of model predictions to empirical data for prevalence against mixed infection rate (blue) and contribution of sib-infection to mixed infections (orange).  Point shape distinguishes African (crosses) and Asian (circles) populations.  The sib-infection fraction is approximated by the relative contribution to the multiply-infected host state (per unit time), at equilibrium, from uninfected hosts relative to singly-infected hosts.  Parameter values (GIVE PARAM VALUES) were chosen to fit with empirical observations.}
  \label{fig:model}
\end{figure}


\section{Discussion}

4. Wider implications of IBD - networks within populations revealing transmission chains.
5. Open problems (impact of sampling strategy, other metrics of diversity, more realistic models).


Genomic data of malaria parasites obtained from human hosts has revealed many striking features of malaria evolution and epidemiology, such as the repeated origin of artemisinin resistance mutations in South East Asia [REF], high levels of population differentiation [REF] and the presence of many loci that show evidence for adaptive evolution or long-term balancing selection [REF].  However, the extent to which such data can give high resolution information about malaria epidemiology, in both space and time, and thus inform control strategies, is largely unknown.  Here, we have focused on the problem of learning about mixed infections from genome-sequencing data.  Mixed infections have been reported previously [REFS].  However, the tools developed here enable estimates, not only of the presence of multiple strains, but their proportions, relatedness and haplotypes.  We have shown that, in the populations studied here, that between 16\% and 63\% of isolates have multiple infections, and up to 54\% have more than two.  Perhaps most unexpected, we have also shown that over half of all mixed infections include strains related at the level of siblings, with chromosomal patterns of IBD that indicate progeny from a single infected mosquito.  Even in regions of extremely low average prevalence, such as in parts of South East Asia (for example Vietnam has a prevalence of less than 1 in 1,000 overall and even higher transmission regions have rates not much higher \citep{who2017, who2017profile}), where individuals infected with malaria are found, they frequently carry mixed infection (we estimate 35\%) and, again, often with sibling strains (75\% for Vietnam).

To better understand how observations of characteristics of mixed infection relate to epidemiological processes, we introduced a simple deterministic model for mixed infection.  For a given set of transmission parameters (those relating to the relative chances of transmitting single and multiple strains at host-vector interactions), as vector density increases, so does prevalence and, in a roughly linear manner, the expected mixed infection rate.  Consequently, the order of magnitude differences in prevalence seen between Asian and African populations would be expected to translate into orders of magnitude differences in mixed infection rate. That this is not observed points either to differences in the biology of transmission (for example resulting from differences in vector biology) or to heterogeneity of transmission within Asian populations, with hotspots of transmission (in both space and possibly time). Transmission hotspots are highly seasonal \citep{Ahmed2013}. In both Africa \citep{Bejon2010} and Asia \citep{Heng2017}, researchers have identified stable hotspots last for several years and unstable hotspots that disappears after one year or two. Nevertheless, within African populations mixed infection rate is a significant predictor of country-level prevalence, suggesting that the metric is a potentially useful source of information for measuring local transmission intensity.  We note that Senegal, where there has been a dramatic reduction in malaria prevalence \citep{Daniels2015}, is an outlier in terms of this metric within Africa (Figure~\ref{fig:mixInfPlot}).

While many individuals with malaria carry mixed infections, in the majority of cases the pathogen strains are related, often at the level of siblings or even greater.  That sibling strains are frequently observed (we estimate XX \% of all mixed infections) indicates that the transmission bottleneck from vector to host is relatively weak.  One route by which \'supersib\' infections could occur (i.e. sibs from parents that are themselves related) is through onward transmission of sibling strains in a host to a vector, with subsequent meiosis and transmission to a new host. Thus a single mosquito that acquired two distinct malaria strains through biting multiple infected hosts could potentially generate a chain of mixed infections, with increasing levels of IBD.  Such events are likely to play an important role particularly in low transmission settings, which is consistent with our observations of higher levels of IBD in Asian populations and the modelling results.



Limitations.

Finally, ....

\begin{figure}[h]
  \centering{}
  \includegraphics[width=\textwidth]{Fig7.pdf}
  \caption{PLACE HOLDER! NEEDS IMPROVEMENT!}
  \label{fig:NNplot}
\end{figure}




\section{Acknowledgements}
We thank the Pf3k consortium for valuable insights. The project is funded by  Wellcome (100956/Z/13/Z to GM, \ldots).


\section{Data availability}
Metadata on samples is available from \url{ftp://ngs.sanger.ac.uk/production/pf3k/release_5/pf3k_release_5_metadata_20170804.txt.gz}.  Sequence data (aligned to Plasmodium falciparum strain 3D7 v3.1 reference genome sequences, for details see \url{      ftp://ftp.sanger.ac.uk/pub/project/pathogens/gff3/2015-08/Pfalciparum.genome.fasta.gz}) is available from  \url{ftp://ngs.sanger.ac.uk/production/pf3k/release_5/5.1/}.  {\it P. falciparum} prevelance estimates were taken from \url{https://map.ox.ac.uk/country-profiles/#!/}.  Inferred haploytpes from each sample are available from XXX.


\section{Disclosure Declaration}
None declared.


\bibliography{mixedIBD.bib}


\end{document}
